lib/components/viewers/pattern-row.js

```diff
@@ -145,3 +165,5 @@ export default class PatternRow extends Component {
     )
   }
 }
+
+export default connect()(injectIntl(PatternRow))
```

Comment:
Remove `connect` and its import.

Reply:
Removed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/2ccefb263ad8863edab72e9d796596beda7336f0